libs/remix-ui/editor/src/lib/remix-ui-editor.tsx

```diff
@@ -102,8 +102,11 @@ export const EditorUI = (props: EditorUIProps) => {
     const yellowColor = formatColor('--yellow')
     const pinkColor = formatColor('--pink')
     const locationColor = '#9e7e08'
-    const purpleColor = formatColor('--purple')
+    // const purpleColor = formatColor('--purple')
```

Comment:
maybe remove comments

Reply:
i have it intentianaly. we have not much color variables to use, this one is reserved for future changes.